plot_convehull.r

```diff
@@ -0,0 +1,108 @@
+#' Function for plotting convex hulls.
```

Comment:
can you please delete this file? I'm assuming `plot_convehull.r` inside the `/R` folder is the actual one?